scylla/src/frame/cql_types_test.rs

```diff
@@ -205,20 +205,23 @@ async fn test_naive_date() {
 
     let tests = [
         // Basic test values
-        ("0000-1-1", Some(NaiveDate::from_ymd(0000, 1, 1))),
+        ("0000-01-01", Some(NaiveDate::from_ymd(0000, 1, 1))),
         ("1970-01-01", Some(NaiveDate::from_ymd(1970, 1, 1))),
         ("2020-03-07", Some(NaiveDate::from_ymd(2020, 3, 7))),
-        ("1337-4-5", Some(NaiveDate::from_ymd(1337, 4, 5))),
-        ("-1-12-31", Some(NaiveDate::from_ymd(-1, 12, 31))),
+        ("1337-04-05", Some(NaiveDate::from_ymd(1337, 4, 5))),
+        ("-0001-12-31", Some(NaiveDate::from_ymd(-1, 12, 31))),
         // min/max values allowed by NaiveDate
-        ("-262144-1-1", Some(min_naive_date)),
-        ("262143-12-31", Some(max_naive_date)),
+        // NOTICE: dropped for Cassandra 4 compatibility
+        //("-262144-1-1", Some(min_naive_date)),
+        //("262143-12-31", Some(max_naive_date)),
+
         // 1 less/more than min/max values allowed by NaiveDate
         ("-262145-12-31", None),
-        ("262144-1-1", None),
+        // NOTICE: dropped for Cassandra 4 compatibility
+        //("262144-01-01", None),
         // min/max values allowed by the database
         ("-5877641-06-23", None),
-        ("5881580-07-11", None),
+        //("5881580-12-31", None),
```

Comment:
Why change the day and month if you are going to comment it out?

Reply:
leftovers from my experiments with Cassandra 4, will revert to original values